src/components/dev-hub/learn-tabs.js

```diff
@@ -0,0 +1,98 @@
+import React, { useCallback, useContext, useEffect, useState } from 'react';
+import { css } from '@emotion/react';
+import styled from '@emotion/styled';
+import { Tabs as LeafyTabs, Tab } from '@leafygreen-ui/tabs';
+import Button from './button';
+
+const TabButtonStyle = () => css`
+    border-radius: 0;
+    :active,
+    :hover,
+    :focus {
+        color: ${({ theme }) => theme.colorMap.darkGreen};
+        :after {
+            background-color: ${({ theme }) => theme.colorMap.darkGreen};
+        }
+    }
+`;
+
+const StyledTabButton = styled(Button)`
+    ${TabButtonStyle}
+`;
+
+const TabsStyle = () => css`
+    @media (min-width: 1048px) {
+        button:first-child {
+            margin-right: 73%;
+        }
+    }
+    button[aria-selected='true'] {
+        color: ${({ theme }) => theme.colorMap.darkGreen};
+        :after {
+            background-color: ${({ theme }) => theme.colorMap.darkGreen};
+        }
+    }
+`;
+
+const StyledTabs = styled(LeafyTabs)`
```

Comment:
One other thought here is for mobile we could stack if we wanted by adding `flex-direction: column`

Reply:
Mike said he doesn't like the stacking approach we currently have. He prefers a drop down( which is his ideal choice) , if not he is okay with them lining next to each other.

Reply:
Oh okay I see, let's start with them next to each other as we have here and then next step would be a dropdown